triples-generator/src/test/scala/ch/datascience/triplesgenerator/reprovisioning/TriplesVersionCreatorSpec.scala

```diff
@@ -30,23 +30,30 @@ import org.scalatest.wordspec.AnyWordSpec
 
 class TriplesVersionCreatorSpec extends AnyWordSpec with InMemoryRdfStore with should.Matchers {
 
-  "insertCliVersion" should {
+  "updateCliVersion" should {
 
-    "create a renku:CliVersion object with the given version" in new TestCase {
+    "create/update a renku:CliVersion object with the given version" in new TestCase {
       findVersionInDB shouldBe Set.empty
 
-      creator.insertCliVersion().unsafeRunSync()
+      creator0.updateCliVersion().unsafeRunSync()
```

Comment:
Maybe it would be a bit nicer if we'd have a named factory for the Creators like a `def newCreator(version: CliVersion)` so we can do `newCreator(version = currentCliVersion).updateCliVersion().unsafeRunSync()`.